Automattic/wp-calypso
# Start With Square: Chores

## Proposed Changes

* Remove Square Flag check for Start With general behaviors
* Move style file to inside the partner folder


## Why are these changes being made?
* The flag behaviors will not depend on this specific partner
* The other partners landing pages will have a different layout so the general style shouldn't be applied to all pages anymore 


## Testing Instructions

* Go to `/start-with/square-payments`
* Everything should be working similarly as in production 
* Go to a route with a not existent partner as `/start-with/another-partner`
* You should a not found page
